packages/offix-cache/src/createMutationOptions.ts

```diff
@@ -34,6 +39,14 @@ export interface MutationHelperOptions<T = {
    * For example for `modifyObject(value: String!): Object` value will be `Object`
    */
   returnType?: string;
+
+  /**
+   * [Modifier]
+   *
+   * Maps input objects for the cases if variables are not passed to the root
```

Comment:
It is hard to understand it. I think an example will work better than anything else.

Reply:
We can also mention that there is traverse mapper that will flatten your input types